bindgen/spec.yml

```diff
@@ -1,5 +1,3 @@
-# yaml-language-server: $schema=generated/spec.schema.json
```

Comment:
This should not be removed, as it allows editor support to lint and provide auto-completion.